test/integration/master/synthetic_master_test.go

```diff
@@ -224,6 +225,101 @@ func TestStatus(t *testing.T) {
 		}
 	}
 }
+func constructBodyByManyLabels(val string, size int) string {
+
+	var temp = `{
+		"apiVersion": "apps/v1beta1",
+		"kind": "Deployment",
+		"metadata": {
+		   "name": "test-deployment2",
+		   "namespace": "default",
+		   "labels": {`
+
+	for i := 0; i < size; i++ {
+		temp += "\"test" + strconv.Itoa(i) + "\" : \"" + val + "\" ,"
+	}
+
+	temp += `"test100000" :"a" 
+			}
+			},
+			"spec": {
+			"replicas": 1,
+			"template": {
+			"metadata": {
+				"labels": {
+				"foo": "bar"
+				}
+			},
+			"spec": {
+				"containers": [{
+				"name": "foo",
+				"image": "foo"
+				}]
+			}
+			}
+			}
+			}`
+
+	return temp
+}
+
+func TestObjectSizeResponses(t *testing.T) {
+
+	_, s, closeFn := framework.RunAMaster(nil)
+	defer closeFn()
+	transport := http.DefaultTransport
+
+	var expectedMsgFor1MB = `etcdserver: request is too large`
```

Comment:
we usually use `:=` instead of `var` for those constants.

Reply:
Updated